### ISSUE TITLE
New Techniques

### ISSUE THREAD

**Issue by szafarnia** [HumanBrainProject/openMINDS_controlledTerms#391]
Dear all,
Could following techniques be registered in KG?
- temporal mean
- dhollander response function estimation
- fibre orientation distribution (FOD) estimation
- streamlines tractography
- spherical-deconvolution informed filtering of tractograms (SIFT): (paper ref: https://www.sciencedirect.com/science/article/pii/S1053811912011615)

**Comment by lzehl** [comment]
@szafarnia thank you for your request. I finally found some time to do a bit of research for your technique suggestions.

I think the following:

**temporal mean** is the measurement/result of an analysis technique which I would probably register as **arithmetic average analysis** which was just performed of a series of values over a time period. @mdenker what do you think? is the setting special enough to define something like **temporal arithmetic average analysis** ? (might be, I need your feedback)

**dhollander response function estimation** is a specific **response function estimation technique** which seems to be implemented under the name "dhollander" in [MRtrix](https://mrtrix.readthedocs.io/en/dev/reference/commands/dwi2response.html). It seems to be that Dhollander actually published two different versions of his implementation of an RF estimation technique. @mdenker @skoehnen what do you think?  I tend to register **response function estimation technique** first. Maybe we can consider **Dhollander's response function estimation technique**  in addition?

**fibre orientation distribution (FOD) estimation** I think this could be registered as **fibre orientation distribution estimation technique**. @skoehnen I would need help to formulate a definition here.

**streamlines tractography** should be just registered as "tractography" (I don't think that there is a difference) @skoehnen would you agree?

**spherical-deconvolution informed filtering of tractograms** should be probably **spherical-deconvolution informed filtering** @skoehnen would you agree?

**Comment by skoehnen** [comment]
> **dhollander response function estimation** is a specific **response function estimation technique** which seems to be implemented under the name "dhollander" in [MRtrix](https://mrtrix.readthedocs.io/en/dev/reference/commands/dwi2response.html). It seems to be that Dhollander actually published two different versions of his implementation of an RF estimation technique. @mdenker @skoehnen what do you think? I tend to register **response function estimation technique** first. Maybe we can consider **Dhollander's response function estimation technique** in addition?

I agree, we should register both, the more generic **response function estimation technique** and the **Dhollander's response function estimation technique**

> **fibre orientation distribution (FOD) estimation** I think this could be registered as **fibre orientation distribution estimation technique**. @skoehnen I would need help to formulate a definition here.

I need to research a bit the difference to ODFs.

> **streamlines tractography** should be just registered as "tractography" (I don't think that there is a difference) @skoehnen would you agree?

Tractography should be fine.

> **spherical-deconvolution informed filtering of tractograms** should be probably **spherical-deconvolution informed filtering** @skoehnen would you agree?

Yes, I agree.

**Comment by lzehl** [comment]
further research from @skoehnen :
fibre orientation distribution estimation (FOD estimation) | fibre orientation distribution function (fODF) [seems to be synonyms]
Remaining questions for this one: is this really fibre specific or can the analysis technique work on anything with orientation distribution? @skoehnen we need to further investigate this.

**Comment by lzehl** [comment]
For sure: orientation distribution function / orientation distribution estimation are super classes of these ones

**Comment by lzehl** [comment]
The spherical-deconvolution informed filtering actually refers to Constrained Spherical Deconvolution

**Comment by skoehnen** [comment]
Here the references we used during the discussion:

- https://mrtrix.readthedocs.io/en/latest/reference/commands/dwi2fod.html
- https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6585735/
- https://www.sciencedirect.com/science/article/pii/S0010482521004376#bib6
- https://en.wikipedia.org/wiki/Spherical_harmonics
